backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/pporder/process/WEBUI_PP_Order_PrintLabel.java

```diff
@@ -98,24 +98,30 @@ protected String doIt()
 		final DocumentIdsSelection selectedRowIds = getSelectedRowIds();
 
 		final ImmutableList<PPOrderLineRow> selectedRows = getView().streamByIds(selectedRowIds)
+				.filter(ppOrderLineRow -> ppOrderLineRow.getType().isHUOrHUStorage())
```

Comment:
if u are filtering here by HU or HUStorage type.... below you won't get any MainProduct row...